extensions/project-launcher/package.json

```diff
@@ -0,0 +1,169 @@
+{
+  "$schema": "https://raw.githubusercontent.com/vicinaehq/vicinae/refs/heads/main/extra/schemas/extension.json",
+  "name": "project-launcher",
+  "title": "Project Launcher",
+  "description": "Open existing or new projects in your preferred editor.",
+  "categories": ["Productivity"],
+  "license": "MIT",
+  "author": "benzipperer",
+  "contributors": [],
+  "pastContributors": [],
+  "icon": "extension_icon.png",
+  "commands": [
+    {
+      "name": "code-in-new-folder",
+      "title": "Code in New Folder",
+      "subtitle": "Create and open a project folder",
+      "description": "Creates a folder in your configured base directory and opens it in your preferred code editor",
+      "mode": "view"
+    },
+    {
+      "name": "search-recent-projects",
+      "title": "Search Recent Projects",
+      "subtitle": "Find and open recent projects",
+      "description": "Search and open recently accessed projects from your preferred editor's workspace history",
+      "mode": "view"
+    }
+  ],
+  "preferences": [
+    {
+      "type": "dropdown",
+      "name": "programName",
+      "title": "Preferred Editor",
+      "description": "Select the editor to open projects with",
+      "default": "positron",
+      "required": true,
+      "data": [
+        {
+          "title": "Positron",
+          "value": "positron"
+        },
+        {
+          "title": "VS Code",
+          "value": "code"
+        },
+        {
+          "title": "Cursor",
+          "value": "cursor"
+        }
+      ]
+    },
+    {
+      "type": "dropdown",
```

Comment:
We have the `runInTerminal` API for this: https://api-reference.vicinae.com/runInTerminal.html
You should use this instead, and remove the preference